src/run/parser/composite.rs

```diff
@@ -23,6 +23,8 @@ pub struct ParsedRun {
 }
 
 fn parsed(run: Run, kind: TimerKind) -> ParsedRun {
+    let mut run = run;
+    run.fix_splits();
```

Comment:
I wonder if this is the appropriate way to fix the splits when "opening them". Maybe that's something LiveSplit One should do instead, or maybe whenever the Timer sets a Run? (cc @wooferzfg)

Reply:
I was thinking that or possibly when they're done being parsed, in the actual parse function, however that would have the same result as this in the end. But having it in livesplit-core would solidify that they would be fixed regardless of implementation.

Reply:
Yeah, I think it makes more sense for LSO to do it. Also, run.fix_splits() needs to be called before saving splits as well; I'm not sure if we do that already or not.

Reply:
If livesplit-core handled fixing splits when parsing and before saving that might work well too.